bookkeeper.py

```diff
@@ -1,58 +1,103 @@
 import numpy as np
-from preprocess_observation import preprocess_observation
 import pickle
-#https://stackoverflow.com/questions/1499119/python-importing-package-classes-into-console-global-namespace
 from game_model import *
```

Comment:
I'd change this to:

    import game_model as gm

and then change your constants (e.g. change OFFSET_HEALTH to gm.OFFSET_HEALTH) in order to both reduce IDE warnings and make it clear where the constants are coming from.